tests/acceptance/features/apiConfigReport/occConfigReportReturnCode.feature

```diff
@@ -0,0 +1,9 @@
+@api
+Feature: generate a configreport via occ
+  As an administrator
+  I want to generate a configreport
```

Comment:
why would I want one ? maybe add "to file a support / help request".
I don't think the main purpose is to check the server report but more to be able to get help on central / github / support